### PR TITLE
docs: Updated documentation according with latest webui changes

### PR DESCRIPTION
**Pull request by pierantoniomerlino**
This PR removes useless entries in the Asset page.

### CODE REVIEW

File: docs/connect-field-devices/asset-implemetation.md [hunk 1]
```diff
@@ -56,9 +56,7 @@ The Asset supports applying a scale and offset to the values obtained by the att
 The following **mode**s are implemented for computing scale and offset:
```
Comment (pierantoniomerlino) [inline docs/connect-field-devices/asset-implemetation.md]: IMO, the term **mode** here is a bit misleading. Is it better to present her the **scaleoffset.type** instead?

Comment (salvatore-coppola) [reply docs/connect-field-devices/asset-implemetation.md]: with @nicolatimeus we decided to adopt this name, let see what he thinks about.

**Comment by salvatore-coppola** [comment]
@pierantoniomerlino i'd change the documentation, now that we have the string in metatype we can add coherent support also for float and integer.

**Comment by salvatore-coppola** [comment]
@nicolatimeus pier is a co-author, we need a third person to approve this.

**Review by salvatore-coppola — DISMISSED** [review]

**Comment by salvatore-coppola** [comment]
Merge after #5584

File: docs/connect-field-devices/asset-implemetation.md [hunk 2]
```diff
@@ -84,29 +84,26 @@ If `channel_value` is 5 (INTEGER) the result is: `(int) ((double) channel_value
...
 - **scale**: 3.25
 - **offset**: 1.5
 
 If `channel_value` is 5 (DOUBLE) the result is: `(double) (channel_value * (double) 3.25d + (double) 1.5d) = 17.75`.
 
-!!! warning
-    If the **mode** is `LONG` or `INTEGER` the decimal part of the **scale** and **offset** value will be discarded.
-
-As the examples show the final result can be different depending on the used **mode** and **value.type**.
+As the examples show, the final result can be different depending on the used **mode** and **value.type**.
```
Comment (mattdibi) [inline docs/connect-field-devices/asset-implemetation.md]: Wording

```suggestion
As the examples show, the final result can change depending on the **mode** and **value.type** used.
```

File: docs/connect-field-devices/asset-implemetation.md [hunk 3]
```diff
@@ -164,4 +161,4 @@ If, for example, the user downloads the channels list following the steps descri
 
 ### Replace current channels
 
-The `Replace current channels` must be checked when the user wants to replace **all** the channels currently present in the asset, with the ones that will be created by the csv uploading.
\ No newline at end of file
+The `Replace current channels` must be checked when the user wants to replace **all** the channels currently present in the asset, with the ones that will be created by the csv uploading.
```
Comment (mattdibi) [inline docs/connect-field-devices/asset-implemetation.md:164]: Is this more readable? Feel free to ignore this if not.

```suggestion
Checking `Replace current channels` when uploading a `csv` file will result in **all** the channels in the asset being replaced by the `csv` file content.
```

**Review by mattdibi — APPROVED** [review]

**Pull request merged**